substrate/client/transaction-pool/src/single_state_txpool/single_state_txpool.rs

```diff
@@ -742,7 +742,12 @@ where
 				});
 			}
 
-			pool.resubmit_at(&hash_and_number, resubmit_transactions).await;
+			pool.resubmit_at(
+				&hash_and_number,
+				resubmit_transactions,
+				ValidateTransactionPriority::Submitted,
```

Comment:
dq: Given these transactions are resubmitted as part of the `handle_enactment`, which is called in `maintain`, shouldn't these have?

```suggestion
				ValidateTransactionPriority::Maintained,
```

Reply:
yes, but I don't want to change any flow in old pool (or at least keep the changes minimal). Sending to the other queue can affects some timings - so I'd prefer to keep the old mechanics here.